holidays/locale/en_US/LC_MESSAGES/LA.po

```diff
@@ -0,0 +1,177 @@
+# Laos holidays en_US localization.
+# Authors: PPsyrius <ppsyrius@ppsyrius.dev>, (c) 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Python Holidays 0.34\n"
+"POT-Creation-Date: 2023-09-11 15:10+0700\n"
+"PO-Revision-Date: \n"
+"Last-Translator: PPsyrius <ppsyrius@ppsyrius.dev>\n"
+"Language-Team: Python Holidays localization team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.3.2\n"
+
+#. %s (in lieu).
+#, c-format
+msgid "ພັກຊົດເຊີຍ%s"
+msgstr "%s (in lieu)"
+
+#. Lao New Year's Day (in-lieu).
+msgid "ພັກຊົດເຊີຍບຸນປີໃໝ່ລາວ"
+msgstr "Lao New Year's Day (in-lieu)"
```

Comment:
It seems we'll need to unify "in-lieu" / "in lieu" usage later

Reply:
They're now all unified under "in lieu"